Resources/Prototypes/Floof/Entities/Objects/Weapons/Melee.yml

```
@@ -91,3 +91,95 @@
   - type: BatterySelfRecharger
     autoRecharge: true
     autoRechargeRate: 40
+
+- type: entity
+  name: Halberd
+  parent: UnholyHalberd
+  id: Halberd
+  description: A polarm with an axe blade topped with a spike mounted on a long shaft. Produced by SESWC!
```

Prototype naming - will require changing other references
```suggestion
- type: entity
  name: halberd
  parent: UnholyHalberd
  id: WeaponHalberd
  description: A polarm with an axe blade, topped with a spike, mounted on a long shaft. Produced by SESWC!
```